mkt/extensions/views.py

```diff
@@ -70,8 +70,22 @@ def create(self, request, *args, **kwargs):
         return Response(serializer.data, status=status.HTTP_202_ACCEPTED)
 
     def validate_upload(self, file_obj):
```

Comment:
I'd move all this code to a separate file, like `mkt/extensions/validator.py`, with one entry point function (`validate(file_obj)`). Otherwise it's going to grow and grow and grow, and it's already starting to pollute the views.